apps/server/src/controllers/integrationController.ts

```diff
@@ -48,62 +48,39 @@ const actionHandlers: Record<string, ActionHandler> = {
     return { payload: updatedEvent };
   },
   /* Message Service */
-  'set-timer-message': (payload) => {
-    if (!isPartialTimerMessage(payload)) {
-      throw new Error('Payload is not a valid timer message');
+  //TODO: maybe coerce the boolean values
```

Comment:
maybe we should, can you explain?
We cant get things of the wrong type entering our inner app logic

Reply:
should we only accept actual boolean values or convert strings and numbers to boolean as well?

Reply:
I think you can decide that. 

I never remember working with Boolean's in OSC, not sure if for lack of availability or just never had a use case for it

I would suggest we model this against the simplest workflow and wait for users feedback if improvement is needed.